File: site-estatico/models/tipo_medicao.js
'use strict';

module.exports = (sequelize, DataTypes) => {
    let tipo_medicao = sequelize.define('tipo_medicao',{	
		id_tipo_medicao: {
			field: 'id_tipo_medicao',
			type: DataTypes.INTEGER,
			primaryKey: true,
			autoIncrement: false
		},	
		fk_componente: {
			field: 'fk_componente',
			type: DataTypes.INTEGER,
			primaryKey: true,
			autoIncrement: false
		},
        tipo: {
			field: 'tipo',
			type: DataTypes.STRING,
			allowNull: true
		},
        unidade: {
			field: 'unidade',
			type: DataTypes.STRING,
			allowNull: true
		},
	}, 
	{
		tableName: 'tipo_medicao', 
		freezeTableName: true, 
		underscored: true,
		timestamps: false,
	});

    return tipo_medicao;
};

File: Apps/Java/clientJava/src/main/java/com/mycompany/projeto/artefato/sprint/entidades/TipoMedicao.java
package com.mycompany.projeto.artefato.sprint.entidades;

public class TipoMedicao {
    private Integer id_tipo_medicao;
    private String tipo;
    private String unidade;

    public TipoMedicao(Integer id_tipo_medicao, String tipo, String unidade) {
        this.id_tipo_medicao = id_tipo_medicao;
        this.tipo = tipo;
        this.unidade = unidade;
    }
}
